### ISSUE TITLE
Minor typos and potentially missing instructions

### ISSUE THREAD

**Issue by wisp92** [Sigourn/morrowind-improved#39]
I found the following minor typos and potentially missing instructions in the guide and I report them here for you to check if they need to be addressed:

1. There is a typo in Watch the Skies: The directory is textures\tew; not textures\tw. (found both in the main.md and the addendum.md)
2. There is a missing instruction in Watch the Skies: The authors of Weather Adjuster and Watch the Skies instruct to turn off cloud handling in Weather Adjuster when using Watch the Skies. (found both in the main.md and the addendum.md)
3. There is a missing instruction in Weapon Sheathing: The Data Files directory must be selected in the installation. (both in the main.md and the addendum.md)
4. The shader mods found in main.md are absent from the Shaders section in the addendum.md.
5. There is a typo in the Shaders configuration: The shader's name is deband_fogawarev3; not deband_fogawarev2. (found in the addendum.md)

Thank you for taking the time to make this guide. If I find anything else I'll add it in this Issue page.

**Comment by LucasDelfino92** [comment]
Thank you! I'll fixed all those issues, minus number 3. I have to check if the MWSE Config file for Weather Adjuster disables cloud handling.

**Comment by LucasDelfino92** [comment]
I can confirm it does disable it, so I can close this issue now. Thank you for your reports!

**Comment by wisp92** [comment]
Your welcome! I found a few typos to report:

1. The entries in the list of hotkeys that correspond to the Quick Equip should map to the E + Left Click combinations; not Shift + E.
2. The Sophisticated Save System also offers a new combination for creating permanent saves through Alt + F5 that might be useful.

**Comment by LucasDelfino92** [comment]
Thanks! Fixed those too.

**Comment by wisp92** [comment]
Speed and Movement Rebalanced sets the fJumpRunMultiplier to less than 1.0, resulting in a setup where a jump while walking is 4 times longer than a jump while running. This ends up being very unrealistic, especially for Khajiit characters that start with increased jumping distance.  qqqbbb posted the issue two years ago, but there was no reply from the author so far. I suggest a value of 1.1 for the fJumpRunMultiplier, but the default of 1.0 works well too.

Some more setting issues I found so far are the following:

- The world map is set to None at the beginning of the game. I am not sure which mod (if any) causes the issue, but currently, the player needs to select the map manually in the Map and Compass settings. This needs to be done only once.
- For beast races, the Character Sound Overhaul calculates the armor sound settings from the weight of the cuirass, while for other races it calculates it from the weight of the boots. For consistency, it might be better to Enable the Alternative Weight Mechanic and reduce the player armor volume to be a bit less intrusive for beast races. 
- One more entry for the list of hotkeys: Speed and Movement Rebalanced adds the sprint functionality on ALT while running.

Are these helpful? I don't want to post personal preferences here. I consider these more like bugs related to the specific mods, but you might think otherwise.

**Comment by LucasDelfino92** [comment]
Thanks for these! I'll look them up when I have some time. For Speed and Movement Rebalanced, the idea would be to simply delete the change to the fJumpRunMultiplier. The intention behind that GMST tweak was for running jumps to not be a viable way to travel through the game.

**Comment by wisp92** [comment]
- Better Quest List has a design flaw regarding the way it handles event registration in the function onJournalEnter. It's possible for an event to register twice during loading. I am currently reading through the code to suggest a solution to the author.
- Ashfall also gives me some errors. I think they are related to loading a savefile after setting up a tent and feature an attempt to access an invalidated item in tentcontroller.lua. However, this mod is massive, so tracking down the bug won't be easy.
EDIT: I contacted Merlord in the Discord and this will be fixed in the upcoming version. It's harmless.
- Morrowind crashed, once so far, and MWSE.log suggests that the fatal error happened to Stealth Improved. However, I don't see any immediate culprits there and further investigation is in order if the crash happens again.

Note that although I am using the Morrowind# (Addendum & BTB), I made the following changes:

1.  I use my mods as replacements for MWSE Clock Block and the MWSE part of Speed and Movement Rebalanced. They are in Beta testing, but I am confident that they didn't cause the issue.
2. I use Lore-friendly trade restrictions instead of MWSE Hide the Skooma, but since by the time of the crash I hadn't visited any merchants or talked with any NPC in general I don't think it is related.
3. I have added Distraction, Talking Matters, Passing Time While Reading, The Levitation Act, and Illegal Summoning; the ones that include a .esp file were checked with tes3view for conflicts and patched accordingly using the constructor. If any of these are related to the crash my best bets would be Distraction and Illegal Summoning, since I was fighting a summoner at that moment. I'll check both there codes just in case.
